Examples/Image/Classification/AlexNet/readlabels.py

```diff
@@ -0,0 +1,85 @@
+import sys
+import time
+from Read_labelclsloc import *
+
+def readtrain(traindir):
```

Comment:
I suggest renaming this to `read_train() `to be consistent with convention. Also applies to other method names elsewhere, such as `readval `below.